modules/core/src/lib/deck.js

```diff
@@ -133,7 +133,7 @@ const defaultProps = {
   onBeforeRender: noop,
   onAfterRender: noop,
   onLoad: noop,
-  onError: null,
+  onError: (error, ...args) => log.error(...args, error)(),
```

Comment:
- This default value breaks the default logging of layer lifecycle errors: https://github.com/visgl/deck.gl/blob/master/modules/core/src/lib/layer-manager.js#L219
- This is an API change but not documented